Client/mods/deathmatch/logic/CClientPed.h

```diff
@@ -788,4 +790,7 @@ class CClientPed : public CClientStreamElement, public CAntiCheatModule
     CClientPed*   m_pGettingJackedBy;                    // The ped that is jacking us
 
     std::shared_ptr<CClientModel> m_clientModel;
+
+    bool m_hasSyncedAnim;
+    bool m_animationOverridedByClient;
```

Comment:
You should zero-initialize these member fields.
```suggestion
    bool m_hasSyncedAnim{};
    bool m_animationOverridedByClient{};
```